File: javascript/javascript_class/person.js
// Javascript class prototype example

function Person(firstName, lastName){
	this.firstName = firstName;
	this.lastName = lastName;
}

Person.prototype = {
	introduce: function(){
		console.log("Hello my name is " + this.firstName + " " + this.lastName);
	}
}

// driver code
me = new Person("Calvin", "Settachatgul");
me.introduce();

File: python/python_class/person.py
# Python class example

class Person:
	def __init__(self, first_name, last_name):
		self.first_name = first_name
		self.last_name = last_name
	
	def introduce(self):
		introduction = "Hello my name is {} {}.".format( self.first_name, self.last_name)
		print( introduction)

me = Person("Calvin", "Settachatgul")

# driver code
print (me)
me.introduce()
